src/miniflask/__init__.py

```diff
@@ -1,6 +1,5 @@
 from .miniflask import miniflask as init, print_info, get_default_args
 from .event import outervar
-from .modules import *
```

Comment:
I assume this is leftover from #114 ?

Reply:
Yes, this is the case. We could push this change directly onto master, but the change is only cosmetic, so I left it here.